packages/node_modules/pouchdb-mapreduce-utils/README.md

```diff
@@ -1,21 +1,20 @@
-pouchdb
```

Comment:
this README accidentally had a lot of typos in it. I corrected it as part of this PR.